### ISSUE TITLE
Writing JSON in pretty print format.

### ISSUE THREAD

**Issue by wagnerwj** [ywwbill/API-LORELEI#1]
have you used a Jackson ObjectMapper and writerWithDefaultPrettyPrinter ?
https://www.mkyong.com/java/how-to-enable-pretty-print-json-output-jackson/

**Comment by ywwbill** [comment]
I am aware that some of the Java packages could print JSON objects in a pretty format. I'm currently using Gson which does support pretty print. The problem is at the transmission of the JSON String. The JSON String is in pretty format when it is created, but all the spaces and newlines are removed after transmitted from the server to the client. That's why I suggest the users use a third-party JSON tool on the client machine. If you have a better solution, please let me know. Thanks!

**Comment by wagnerwj** [comment]
No, your analysis is good; the receiving service should have the job of how
to use the output.
